src/saltext/github/modules/github.py

```diff
@@ -1886,4 +1897,318 @@ def _query(
             # Last page already processed; break the loop.
             next_page = False
 
-    return complete_result
+    return ret
+
+
+def _param_dict(rule_params=False, rule_id=False, page=False, **kwargs):
```

Comment:
**question (non-blocking):** this is interesting. does the GitHub API expect all keyword arguments to be present, even if they're not being used/set? if we were just using this internally, I'd say it was a good use case for a `defaultdict` (something like `foo = defaultdict(lambda: None)`)